package-lock.json

```diff
@@ -27781,6 +27781,7 @@
     },
     "node_modules/npm/node_modules/lodash._baseindexof": {
       "version": "3.1.0",
+      "extraneous": true,
```

Comment:
It looks like the package lock file needs to be updated.  After I ran:
```
git clean -xfd && npm install && npm run compile
```

...package-lock.json had been modified.
in the branch's version, in some of the lodash modules, there are a few `"extraneous": true,` which aren't in my version after I built everything from scratch.  I'm not sure what's going on...